src/uniprotkb/components/entry/DiseaseAndDrugsSection.tsx

```diff
@@ -22,13 +22,18 @@ const DiseaseAndDrugsSection: FC<{
   data: UIModel;
   primaryAccession: string;
   sequence: string;
-}> = ({ data, primaryAccession, sequence }): JSX.Element | null => {
+  taxId: number | undefined;
+}> = ({ data, primaryAccession, sequence, taxId }): JSX.Element | null => {
   if (!hasContent(data)) {
     return null;
   }
+
+  const taxonomyBasedSection =
+    taxId === 9606 ? EntrySection.DiseaseAndDrugs : EntrySection.Phenotypes;
+
   return (
-    <div id={EntrySectionIDs[EntrySection.DiseaseAndDrugs]} data-entry-section>
-      <Card title={EntrySection.DiseaseAndDrugs}>
+    <div id={EntrySectionIDs[taxonomyBasedSection]} data-entry-section>
```

Comment:
Following previous comment.
So that here you would call the function with the tax id here for example and decide what title to generate